components/ui/TypedWord.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="typed-word__container" :class="{ 'typed-word__cursor': showCursor }">
+    <span class="typed-word__content">{{ value }}</span>
+  </div>
+</template>
+
+<script>
+// Implementation based on https://github.com/webnoobcodes/vuejs-typeeffect
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {
+  @Prop({ type: String, default: '' }) initialWord
```

Comment:
Add TypeScript annotations such as:
```suggestion
  @Prop({ type: String, default: '' }) initialWord: string = ''
```

Reply:
@delapuente - I'm running into another prop mutation issue, when adding the Ts annotation here.

<img width="1680" alt="Screen Shot 2020-09-17 at 7 19 16 AM" src="https://user-images.githubusercontent.com/6276074/93476215-c91bcc80-f8b6-11ea-967a-242b7180e4ed.png">


and the only place it's being referred to is here:

```js
created () {
  this.value = this.initialWord
  setTimeout(this.eraseText, this.newTextDelay + 200)
}
```

any thoughts?

Reply:
(actually, I think I fixed it)